src/pages/languages.js

```diff
@@ -47,7 +48,10 @@ const Languages = ({location}: Props) => (
       <div css={sharedStyles.articleLayout.container}>
         <div css={sharedStyles.articleLayout.content}>
           <Header>Languages</Header>
-          <TitleAndMetaTags title="React - Languages" />
+          <TitleAndMetaTags
+            canonicalUrl={`${urlRoot}/languages`}
```

Comment:
Can you add the trailing slashes to this and `/versions/`? To match the URL you get redirected to if you visit /languages.

Otherwise I think this looks good.

Reply:
Sure! No problem at all, I see your point ;)